pkgs/tools/misc/cf-terraforming/default.nix

```
@@ -0,0 +1,27 @@
+{ buildGoModule, fetchFromGitHub, lib, ... }:
+
+buildGoModule rec {
+  pname = "cf-terraforming";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = pname;
```

Reads a bit nicer I think:
```suggestion
    repo = "cf-terraforming";
```